### PR TITLE
Update Text Classification to Data Classification

### LINKED ISSUE #25921: Rename instances of Text Classification to Data Classification
Rename instances of Text Classification to Data Classification to align with changes in 

dotnet/machinelearning-modelbuilder#1710

### PR DESCRIPTION
**Pull request by jwood803**
Update instances of "Text Classification" to "Data Classification"

Fixes #25921

Preview links:
- [What is Model Builder](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-model-builder?branch=pr-en-us-26027)
- [Loading Data with Model Builder](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-model-builder?branch=pr-en-us-26027)
- [Health Violations Tutorial](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/health-violation-classification-model-builder?branch=pr-en-us-26027)

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla I believe this is ready for review

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Model Builder only accepts TSV, CSV, and TXT files with comma, tab, and semi-col
 Model Builder helps you create models for the following machine learning scenarios:
 
 - Sentiment analysis (binary classification): Classify textual data into two categories.
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md:21]: @jwood803 correct me if I'm wrong but I'm not sure if Sentiment Analysis is an option. In which case we might want to remove it from this list.

**Review by luisquintanilla — COMMENTED** [review]
Looks good to me. Minor nit.

**Comment by luisquintanilla** [comment]
@jwood803 since I was already making updates, I've rolled these changes into #26148

**Comment by jwood803** [comment]
Closing since this was covered in #26148

**Pull request closed without merging**